cmd/snap/main.go

```diff
@@ -0,0 +1,42 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2014-2015 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/ubuntu-core/snappy/client"
+	"github.com/ubuntu-core/snappy/dirs"
+)
+
+func main() {
+	dirs.SetRootDir("/")
```

Comment:
Without it dirs.SnapdSocket is ""
On 24 Nov 2015 08:34, "Michael Vogt" <notifications@github.com> wrote:

> In cmd/snap/main.go
> <https://github.com/ubuntu-core/snappy/pull/154#discussion_r45707557>:
>
> > + *
> > + */
> > +
> > +package main
> > +
> > +import (
> > +	"fmt"
> > +	"os"
> > +	"text/tabwriter"
> > +
> > +	"github.com/ubuntu-core/snappy/client"
> > +	"github.com/ubuntu-core/snappy/dirs"
> > +)
> > +
> > +func main() {
> > +	dirs.SetRootDir("/")
>
> Why is this needed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ubuntu-core/snappy/pull/154/files#r45707557>.
>